ledger/eval.go

```diff
@@ -676,7 +672,7 @@ func (eval *BlockEvaluator) TransactionGroup(txads []transactions.SignedTxnWithA
 
 // prepareEvalParams creates a logic.EvalParams for each ApplicationCall
 // transaction in the group
-func (eval *BlockEvaluator) prepareEvalParams(txgroup []transactions.SignedTxnWithAD) []*logic.EvalParams {
+func (eval *BlockEvaluator) prepareEvalParams(txgroup []transactions.SignedTxnWithAD) ([]*logic.EvalParams, error) {
```

Comment:
My initial understanding was that `prepareEvalParams` will not initialize `EvalParams` for a stateless call (it will just `continue` in L685 below). Is this still true?

Reply:
yes, check data/transactions/verify for logicsig flow